nengo_spa/algebras/hrr_algebra.py

```diff
@@ -82,6 +91,28 @@ def bind(self, a, b):
             raise ValueError("Inputs must have same length.")
         return np.fft.irfft(np.fft.rfft(a) * np.fft.rfft(b), n=n)
 
+    def fractional_bind(self, v, exponent):
+        r = np.fft.ifft(np.fft.fft(v) ** exponent)
```

Comment:
Wouldn't this work with `irfft/rfft`? That would allow to return `r` instead of `r.real` and reduce the amount of operations by half. But would require to change the check below (I guess you need to check the complex angle of the returned Fourier coefficients, but that would more closely relate to the message anyways).